### PR TITLE
Fixes on logging.rst

### PR DESCRIPTION
**Pull request by Seluj78**

### CODE REVIEW

**Review by eamanu — APPROVED** [review]
lgtm. If you don't receive core review, please open a issue on bugs tracker.

**Comment by eamanu** [comment]
I don't understand why Travis fail

File: Doc/howto/logging.rst [hunk 1]
```diff
@@ -326,7 +326,7 @@ favourite beverage and carry on.
 If your logging needs are simple, then use the above examples to incorporate
 logging into your own scripts, and if you run into problems or don't
 understand something, please post a question on the comp.lang.python Usenet
-group (available at https://groups.google.com/forum/#!forum/comp.lang.python) and you
+group (available at `https://groups.google.com/forum/#!forum/comp.lang.python`_) and you
```
Comment (JulienPalard) [inline Doc/howto/logging.rst:329]: There is no need for this to make the link a clickable link, see https://docs.python.org/3/howto/logging.html.

This syntax is usefull if you want to change the wording of the link like:

    `foo <https://>`_

**Comment by csabella** [comment]
Thank you for the suggested changes.  As @JulienPalard pointed out, the link is already clickable, so the suggestion for the link isn't needed.  And in the first change, since `exception` is lowercase, I don't think the markup to the Exception class is needed.  Therefore, I'm going to close this pull request.

**Pull request closed without merging**